qa/statsd/GNUmakefile

```diff
@@ -2,14 +2,50 @@ TOPDIR = ../..
 include $(TOPDIR)/src/include/builddefs
 
 TESTDIR = $(PCP_VAR_DIR)/testsuite/statsd
-RBFILES = $(shell echo *.rb)
+TESTSRCDIR = src
+TESTFILES = $(shell find $(TESTSRCDIR) -type f)
 
 ifeq "$(PMDA_STATSD)" "true"
 default setup default_pcp:
 
-install install_pcp: default $(RBFILES)
+install install_pcp: default $(TESTFILES)
 	$(INSTALL) -m 755 -d $(TESTDIR)
-	$(INSTALL) -m 644 $(RBFILES) $(TESTDIR)
+	# I wish I knew how to extract such listing from $(TESTFILES), not sure why this is needed yet
+	$(INSTALL) -m 644 src/cases/01.py $(TESTDIR)/src/cases/01.py
```

Comment:
In the same way TESTDIR is installed above, each new directory used also needs to be $(INSTALL)'d ... otherwise, the packaging ends up slightly incorrect (nothing 'owns' the directory is its not explicitly installed).

Reply:
I added GNUmakefile's to each qa/statsd dir in 78b11bf (as inspired by qa/slurm). I guess GNUmakefile.install are for post install steps once all test files are `$(INSTALL)`-ed? Will leave only `default setup install clean check` in them since python is interpreted.

Reply:
> I added GNUmakefile's to each qa/statsd dir in [78b11bf](https://github.com/performancecopilot/pcp/commit/78b11bfbc9987b9f97ed2b788ee88ae891abddce) (as inspired by qa/slurm).

Perfect.

> I guess GNUmakefile.install are for post install steps once all test files are `$(INSTALL)`-ed? 

This is the makefile that will be in the pcp-testsuite (rpm/deb) package, used when tests are run from /var/lib/pcp/testsuite